code/numerical_analysis/monte_carlo/src/pi_montecarlo.c

```diff
@@ -6,14 +6,13 @@
 double
 pi_monte_carlo(double point_count)
 {
-    double circle_points, pi, x, y;
-    circle_points = 0;
-    for (int i = 0; i < point_count; ++i)
-    {
+    double pi, x, y, circle_points = 0;
+    int i;
+    for (i = 0; i < point_count; ++i){
         x = (double)random() / RAND_MAX;
         y = (double)random() / RAND_MAX;
         if (x * x + y * y < 1)
-            circle_points++;
+            circle_points += 1;
```

Comment:
Not what I meant. Do ++circle_points instead.

Reply:
Did it.